### PR TITLE
MTL-1864

### PR DESCRIPTION
**Pull request by jpdavis-prof**
# Description
Added instructions to **Pre_Boot_NCN_Image_Configuration.md** on how to use the **csm.ncn.ca_cert** role and how to add that layer to a CFS configuration.

https://jira-pro.its.hpecorp.net:8443/browse/MTL-1864

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

### CODE REVIEW

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 1]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
...
+   "layers": [
+   {
+     "name": "csm-config",
+     "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+     "playbook": "<example-playbook.yml>",
+     "commit": "<git commit id>"
+   },
+   ```
+
+   **NOTE:** There are three existing playbooks availabe for NCN'S
```
Comment (nrockershousen) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:82]:
```suggestion
   **NOTE:** There are three existing playbooks available for NCNs:
```

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 2]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
...
+   - ansible/ncn-worker_nodes.yml
```
Comment (nrockershousen) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:83]:
```suggestion
   - `ansible/ncn-worker_nodes.yml`
```

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 3]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
...
+   - ansible/ncn-storage_nodes.yml
```
Comment (nrockershousen) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:84]:
```suggestion
   - `ansible/ncn-storage_nodes.yml`
```

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 4]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
...
+   - ansible/ncn-master_nodes.yml
```
Comment (nrockershousen) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:85]:
```suggestion
   - `ansible/ncn-master_nodes.yml`
```

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 5]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
...
 
+   **NOTE:** If the platform certificate is needed for the purpose of accessing zypper repos,
+   the `csm.ncn.ca_cert` role can be added to your playbook within the csm-config-management repo.
+
+   ```console
+    # Install the platform certificate
+    - role: csm.ncn.ca_cert
+   ```
+
+   The first layer in the CFS session should be similar to this, where <example-playbook.yml> is the playbook
```
Comment (nrockershousen) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:69]:
```suggestion
   The first layer in the CFS session should be similar to this, where `<example-playbook.yml>` is the playbook
```

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 6]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
...
+   that includes the csm.ncn.ca_cert role.
```
Comment (nrockershousen) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:70]:
```suggestion
   that includes the `csm.ncn.ca_cert` role.
```

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 7]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
 
 1. [Create a CFS Configuration](Create_a_CFS_Configuration.md)
 
+   **NOTE:** If the platform certificate is needed for the purpose of accessing zypper repos,
```
Comment (nrockershousen) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:61]:
```suggestion
   **NOTE:** If the platform certificate is needed for the purpose of accessing Zypper repos,
```

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 8]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
...
+   the `csm.ncn.ca_cert` role can be added to your playbook within the csm-config-management repo.
```
Comment (nrockershousen) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:62]:
```suggestion
   the `csm.ncn.ca_cert` role can be added to a playbook within the `csm-config-management` repo.
```

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 9]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
...
+
+   ```console
+    # Install the platform certificate
+    - role: csm.ncn.ca_cert
+   ```
+
+   The first layer in the CFS session should be similar to this, where `<example-playbook.yml>` is the playbook
+   that includes the `csm.ncn.ca_cert` role.
+
+   ```console
```
Comment (rustydb) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:72]: This should be `json`.

File: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md [hunk 10]
```diff
@@ -58,6 +58,32 @@ a Ceph image.
 
 1. [Create a CFS Configuration](Create_a_CFS_Configuration.md)
 
+   **NOTE:** If the platform certificate is needed for the purpose of accessing Zypper repos,
+   the `csm.ncn.ca_cert` role can be added to a playbook within the `csm-config-management` repo.
+
+   ```console
```
Comment (rustydb) [inline operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md:64]: This looks like it should be `yaml`.

**Review by rustydb — CHANGES_REQUESTED** [review]

**Pull request merged**